File: Basic Programs/javascript/fibonacci.js
const n = 10;
let firstnum = 0;
let secondnum = 1;
for (let index = 0; index <= n; index++) {
   console.log(firstnum + " ");
   const next = firstnum + secondnum;
   firstnum = secondnum;
   secondnum =next;
    
} 

File: Basic Programs/java/fibonacci.java
import java.util.Scanner;

public class fibonacci {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter how many numbers want to print ");
        int fibonacci = scanner.nextInt();
        int firstnum = 0;
        int secondnum = 1;
        for (int i = 0; i <= fibonacci; i++) {
            System.out.println(firstnum + " ");
            int next = firstnum + secondnum;
            firstnum = secondnum;
            secondnum = next;
        }
    
    }
}
